common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -16,9 +16,11 @@
         </span>
     % endif
     </div>
-
-    <fieldset role="${input_type}group" aria-label="${label}">
-
+    % if input_type == 'checkbox':
+        <div class="choicegroup-choices" role="group" aria-labelledby="${label}">
```

Comment:
@ahsan-ul-haq the attribute here needs to be ```aria-label``` not ```aria-labelledby```
 ```aria-label``` takes a text value while  ```aria-labelledby``` takes an IDref.